src/Microsoft.AspNetCore.SystemWebAdapters/Microsoft.AspNetCore.SystemWebAdapters.csproj

```diff
@@ -66,6 +66,10 @@
     <PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <SharedFiles Remove="Adapters\ISystemWebAdapterRemoteAppBuilder.Shared.cs" />
```

Comment:
Why is this removed?

Reply:
oh VS being helpful.... I'll remove this